node-hide-console-window.cc

```diff
@@ -1,19 +1,32 @@
 #include <node.h>
 #include <Windows.h>
 
+HWND GetTargetWindow()
+{
+    HWND consoleWindow = GetConsoleWindow();
+    HWND parentWindow = GetParent(consoleWindow);
+    if (parentWindow == NULL)
+    {
+        return consoleWindow;
+    }
+    return parentWindow;
+}
+
 void HideConsole(const v8::FunctionCallbackInfo<v8::Value>& args)
 {
-    ShowWindow(GetConsoleWindow(), SW_HIDE);
+    ShowWindow(GetTargetWindow(), SW_HIDE);
+    DWORD processId;
```

Comment:
Variable not referenced anywhere else.

Reply:
You are correct, I believe those lines were left in from a previous attempt to find a workaround